config/rbac/operand_rolebinding.yaml

```diff
@@ -11,3 +11,6 @@ subjects:
   - kind: Group
     name: system:serviceaccounts:external-dns
     namespace: external-dns
```

Comment:
Related to comment on [bundle/manifests/external-dns_rbac.authorization.k8s.io_v1_clusterrolebinding.yaml](https://github.com/openshift/external-dns-operator/pull/134/files#diff-bdce64dd417d9ff279afc3d8e36f6bdf34e7583719d8662860370b8f4600ce9a)